pydis_site/apps/api/viewsets/bot/bumped_thread.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework.mixins import (
+    CreateModelMixin, DestroyModelMixin, ListModelMixin, RetrieveModelMixin
+)
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot import BumpedThread
+from pydis_site.apps.api.serializers import BumpedThreadSerializer
+
+
+class BumpedThreadViewSet(
+    GenericViewSet, CreateModelMixin, DestroyModelMixin, RetrieveModelMixin, ListModelMixin
+):
+    """
+    View providing CRUD (Minus the U) operations on threads to be bumped.
+
+    ## Routes
+    ### GET /bot/bumped-threads
+    Returns all BumpedThread items in the database.
+
+    #### Response format
+    >>> [
+    ...     {
+    ...         'thread_id': "941705627405811793",
+    ...     },
+    ...     ...
+    ... ]
```

Comment:
This works, but it's clumsy. It should be flattened to an array. And why is the ID returned as a string? Or is that just a documentation error?

Reply:
Good point, I'll flatten it. Yea, it's a doc error and will fix.

Would this be the best approach to do so? My django knowledge is limited atm

```py
    def list(self, request: HttpRequest, *args, **kwargs) -> list[int]:
        """Returns all BumpedThread items in the database as a flat list."""
        queryset = self.queryset.values_list("thread_id", flat=True)
        return Response(queryset)
```

Reply:
My first thought is to look into customising the serialiser.

Reply:
Is it possible that we may add a field here in the future, such as `'last_bumped_at'` or `'bumped_by`?

Reply:
https://github.com/python-discord/site/pull/663/commits/3e9557056c06a39c077b76d718eb35b99a365711